articles/quickstart/native/ionic-react/interactive.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Add login to your Ionic React with Capacitor app
+default: true
+description: This tutorial demonstrates how to add user login with Auth0 to an Ionic React & Capacitor application.
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - ionic
+  - react
+  - capacitor
+github:
+  path: react
+contentType: tutorial
+useCase: quickstart
+interactive: true
+files:
+  - files/index
+  - files/login-button
+  - files/logout-button
+  - files/app
+  - files/user-profile
+---
+
+# Add login to your Ionic React with Capacitor app
+
+Auth0 allows you to quickly add authentication and gain access to user profile information in your application. This guide demonstrates how to integrate Auth0 with an Ionic (React) & Capacitor application using the [Auth0 SPA SDK](https://github.com/auth0/auth0-spa-js).
+
+<%= include('../_includes/ionic/_article_intro') %>
+
+<%= include('../_includes/_getting_started', { library: 'Ionic' }) %>
+
+<%= include('../_includes/ionic/_configure_urls') %>
+
+<%= include('../../_includes/_auth0-react-install.md') %>
+
+<%= include('../_includes/ionic/_install_plugins') %>
```

Comment:
Would it make sense to use the `../../_includes/_configure_auth0_interactive.md` here? That one refers to the interactive application configurator that is part of the new design. Users can now setup apps and its settings through that.

Reply:
Makes sense, let me have a look.

Reply:
As mentioned on Slack, not sure I can use this include as I need to use `${account.namespace}` and other expansions but when I do that I get an "unexpected token" error.

Reply:
@frederikprijck I copied the content out of that include into `_callback_urls` partial and modified it slightly for Ionic. Will need to share this with Angular but that's on a separate PR.

Looks like I'm getting the signup widget now too 👍🏻 

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/766403/168257716-7db3e57e-6a76-466e-9ec7-7706aa5da9a3.png">